launch/client.py

```diff
@@ -2951,17 +2951,17 @@ def completions_stream(
         for event in events:
             yield json.loads(event.data)
 
-    def create_fine_tune_job(
+    def create_fine_tune(
         self,
         model: str,
         training_file: str,
         validation_file: Optional[str] = None,
         fine_tuning_method: Optional[str] = None,
         hyperparameters: Optional[Dict[str, str]] = None,
         suffix: str = None,
-    ) -> CreateFineTuneJobResponse:
+    ) -> CreateFineTuneResponse:
         """
-        Create a fine-tuning job
+        Create a fine-tune
```

Comment:
qq: are we sticking with "fine-tune" or "fine tune"

Reply:
https://platform.openai.com/docs/api-reference/fine-tunes

So it looks like they refer to them as "fine-tuning jobs" or "jobs that can fine-tune" in prose. If we're talking about a specific type, then it'd be `FineTune`, but eh.